common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -104,6 +108,9 @@ def B(*args, **kwargs):
 AUTH_ENTRY_LOGIN_API = 'login_api'
 AUTH_ENTRY_REGISTER_API = 'register_api'
 
+# Custom auth entry point used by external software
```

Comment:
I think it would be useful to have a more descriptive comment here (like the one you've already added to `aws.py`) to clarify the form and function of this configuration.